### PR TITLE
Implemented InfluxDB Module

### PR DESCRIPTION
**Pull request by aweberlopes**
Here we have implemented the influxdb module and tested with a local docker instance.

Output influx cli
[example-influx.log](https://github.com/racoon63/bandwidth-monitor/files/4390047/example-influx.log)

### CODE REVIEW

File: lib/bwm/database/influx.py [hunk 1]
```diff
@@ -88,23 +88,21 @@ def insert(self, data):
                   "server_latency": data["server"]["latency"],
                   "server_sponsor": data["server"]["sponsor"],
                   "server_url1": data["server"]["url"],
-                  "server_url2": data["server"]["url2"],
+                  #"server_url2": data["server"]["url2"],
                   "client_country": data["client"]["country"],
                   "client_ip": data["client"]["ip"],
                   "client_isp": data["client"]["isp"],
-                  "client_isp_rating": data["client"]["isprating"],
+                  "client_isp_rating": data["client"]["isp-rating"],
```
Comment (racoon63) [inline lib/bwm/database/influx.py:95]: "isp-rating" is going to be changed later to just "isprating".

File: lib/bwm/database/influx.py [hunk 2]
```diff
@@ -120,10 +118,7 @@ def insert(self, data):
             log.exception(err)
 
         
-
-
         else:
             log.info("Recorded data successfully")
-            log.debug("Object ID: {}".format(identifier.inserted_id))
```
Comment (racoon63) [inline lib/bwm/database/influx.py:127]: Is it possible to get an identifier from influx and add it to the debug log to trace the record?

**Review by racoon63 — COMMENTED** [review]
Seems to be okay for me.

File: lib/bwm/database/influx.py (hunk 2 above)
Comment (aweberlopes) [reply lib/bwm/database/influx.py:127]: Had to clarify how accurate ther unix timestamp should be.

Comment (aweberlopes) [reply lib/bwm/database/influx.py:127]: use timestamp from server as identifier for influx as tag

Comment (racoon63) [reply lib/bwm/database/influx.py:127]: accepted.

**Pull request merged**